app/controllers/api/reviews/votes_controller.rb

```diff
@@ -1,10 +1,9 @@
 class Api::Reviews::VotesController < ApplicationController
-  before_action :authenticate_user!
+  before_action :authenticate_user!, only: :create
 
```

Comment:
Doesn't devise do this for us with `current_user`? I was unable to retrieve a vote without mocking an authenticated user.